Code/Framework/AzCore/AzCore/Debug/Trace.h

```diff
@@ -319,22 +313,50 @@ namespace AZ
             "String used in place of boolean expression for AZ_WarningOnce.",                                                   \
             "Did you mean AZ_WarningOnce("#window", false, \"%s\", "#expression"); ?",                                          \
             "Did you mean AZ_WarningOnce("#window", false, "#expression", "#__VA_ARGS__"); ?");                                 \
-        static bool AZ_CONCAT_VAR_NAME(azWarningDisplayed, __LINE__) = false;                                                                     \
-        if (!AZ_CONCAT_VAR_NAME(azWarningDisplayed, __LINE__))                                                                                    \
+        static bool AZ_CONCAT_VAR_NAME(azWarningDisplayed, __LINE__) = false;                                                   \
+        if (!AZ_CONCAT_VAR_NAME(azWarningDisplayed, __LINE__))                                                                  \
         {                                                                                                                       \
             AZ::Debug::Trace::Instance().Warning(__FILE__, __LINE__, AZ_FUNCTION_SIGNATURE, window, __VA_ARGS__);               \
-            AZ_CONCAT_VAR_NAME(azWarningDisplayed, __LINE__) = true;                                                                              \
+            AZ_CONCAT_VAR_NAME(azWarningDisplayed, __LINE__) = true;                                                            \
         }                                                                                                                       \
     }                                                                                                                           \
     AZ_POP_DISABLE_WARNING
 
-    #define AZ_TracePrintf(window, ...)                                                                            \
-    if(AZ::Debug::Trace::Instance().IsTraceLoggingEnabledForLevel(AZ::Debug::LogLevel::Info))                                 \
+    #define AZ_Info(window, ...)                                                                                   \
+    if(AZ::Debug::Trace::Instance().IsTraceLoggingEnabledForLevel(AZ::Debug::LogLevel::Info))                      \
     {                                                                                                              \
         AZ::Debug::Trace::Instance().Printf(window, __VA_ARGS__);                                                  \
     }
 
+    #define AZ_Trace(window, ...)                                                                                  \
+    if(AZ::Debug::Trace::Instance().IsTraceLoggingEnabledForLevel(AZ::Debug::LogLevel::Trace))                     \
+    {                                                                                                              \
+        AZ::Debug::Trace::Instance().Printf(window, __VA_ARGS__);                                                  \
+    }
```

Comment:
This is going to continue forward with having to put `\n` at the end of every Info/Trace call, which isn't required for Warning/Error/Assert.  I think we should probably take this opportunity to make these two calls to Printf always include a `\n` (probably add another parameter to Printf) so they're consistent with the other 3 APIs.

Reply:
Make the change to add a the newline is not so simple ....

The macro itself would need to create a local fixed_string_buffer to format the `__VA_ARGS__` into before calling Printf.
The easiest solution would be to add a `PrintfWithNewline` function that does so in the context of a function.